syntax = "proto2";
package cardgame;

import "struct.proto";
option optimize_for = LITE_RUNTIME;
option java_package = "com.zhili.cardgame.proto";  
option java_outer_classname = "Chat";



message SendChatReq {
    enum DEF {
        ID = 1200;
    }
    required int32 table_id = 1; // table_id > 0 牌局内聊天
	required string message = 2;
}

message SendChatResp {
    enum DEF {
        ID = 1201;
    }
    optional Result result = 1;
}


message ChatEvent {
	enum DEF {
		ID = 1250;
	}
	required int32 sender = 1;
	required int32 table_id = 2;
	required string message = 3;
}

